GitUp/Application/AppDelegate.m

```diff
@@ -47,16 +47,12 @@ @interface AppDelegate () <NSUserNotificationCenterDelegate, SUUpdaterDelegate>
 @property(nonatomic, strong) WelcomeWindowController *welcomeWindowController;
 @end
 
+#import "KeychainAccessor.h"
```

Comment:
Can you see if `#import <Security/Security.h>` is needed anymore? There might be something else in AppDelegate that needs it?

Reply:
@lucasderraugh 
Yes :) I also delete unnecessary imports in other headers. I would like to change import in `WindowController.h`:

```objective-c
#import <GitUpKit/GitUpKit.h>

@interface WindowController : GIWindowController <NSWindowDelegate>
@end
```

to

```objective-c
#import <GitUpKit/GIWindowController.h>

@interface WindowController : GIWindowController <NSWindowDelegate>
@end
```